index.bs

```diff
@@ -44,6 +44,105 @@ url: https://dom.spec.whatwg.org/#ref-for-concept-getelementsbytagname; type: in
+
+Hyperlinks, links from one page to another, are one of the foundations of the Web.
+
+Following a link, or visiting a web page, should be safe.
+Users who are following a link to a web page they haven't seen before
+should not have to fear for the security of their computer
+or for essential aspects of their privacy.
+(One caveat here is that users of the Web
+should not assume that they are free from consequences
+if their use of the Web is harming others.)
+
+Saying “essential aspects” here skips over quite a bit of detail.
+The Web isn't perfectly private.
+But we can probably make it better than it is today.
+In doing so, we may want to focus on the cases where
+user expectations of what information is revealed when using the Web
+don't match the technical details.
+We might be able to use the design of user interfaces
+to make it clear what users of the Web should expect,
+since one aspect of privacy problems is when reality doesn't meet expectations.
+For example,
+a person walking down the street generally expects to be recognized by their friends,
+but depending on the country,
+may not expect that they were walking down that street to be recorded in a permanent government database.
+Online, people might have less understanding of what is or isn't possible,
+and their expectations might differ more widely.
+<!-- Can we cite research on this??? -->
+
+When we add new features to the Web
+that might weaken the security or privacy characteristics that the Web currently has,
+we should consider the tradeoffs involved in that particular feature.
+A new feature with security risks might still make users safer, for example,
+because they would download and run native binary software less often,
+and downloading and running binaries is much riskier than visiting a Web page.
+However, we should also consider the tradeoff
+against the *idea* that it is safe to visit Web pages:
+adding a risky feature to the Web teaches the Web's users that the Web is less safe,
+and this erodes one of the core values of the Web:  that it is safe to visit websites.
+We should seek to give users a true picture of the safety of the Web
+so that they can act based on that understanding of safety
+(in comparison to the safety of alternatives),
+and at the same time we should seek to keep the Web safe for its users.
+
+<h3 id="trusted-ui">Trusted user interface should be trustworthy</h3>
+
+Since users of the Web can follow links to sites they might not know already,
+software that lets them access the Web generally provides user interface
+to show the user who they're interacting with and how,
+such as by displaying part or all of the URL or other information about the site,
+or whether the connection is secure.
+Since users rely on this information to learn what site they're on
+and to make judgments about whether it is trustworthy,
+it is important that sites not be able to spoof or override this user interface.
```

Comment:
This seems out of scope for API design, albeit a critical point for user agent UI design.

It seems like the point below is the more critical one: if a feature relies on adding extra decisions for a user to make, care should be taken that the feature is providing enough value (either through providing a clear user benefit, or through simplifying some other decision process for the user) to make it worthwhile.

Reply:
I see your point, @alice, but I think it's in scope to the extent that different API shapes make different UI designs easier / more likely / etc.

Reply:
Perhaps that point should be made more explicitly, in that case?

Reply:
Agreed!

Reply:
https://github.com/w3ctag/design-reviews/issues/461#issuecomment-587618596 this seems like a good first pass.

Reply:
I'm a little confused by @alice's last comment in this thread -- was it meant as a comment on the following section (on informed consent), rather than this section on trusted UI?

(Also, I think the "point below" in the initial comment was referring to the second paragraph of this section, and not the following section, but please correct me if I'm wrong.)

In any case, what I've tried to do here (and I realize I should comment before pushing the changes since pushing the changes will likely hide the review comments) in convert these two paragraphs into one and connect them more closely.  Though perhaps you were looking for something different?